android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -157,6 +166,24 @@ public void networkUnavailable() {
 
     }
 
+    private void showProgressBar(boolean show) {
+        if(show)
+            downloadProgressDialog.show();
+        else
+            downloadProgressDialog.dismiss();
+    }
+
+    private void setupProgressBar() {
+        downloadProgressDialog = new ProgressDialog(getContext());
+        downloadProgressDialog.setIndeterminate(true);
+        downloadProgressDialog.setProgressPercentFormat(null);
+        downloadProgressDialog.setProgressNumberFormat(null);
```

Comment:
Oh yes. By default the progress bar shows the percentage and the number in the progress. Hence it always stays 0/100 since its a binary task. Hence had to explicitly remove it.